daemon/logger/journald/journald.go

```diff
@@ -48,10 +49,17 @@ func New(ctx logger.Context) (logger.Logger, error) {
 		name = name[1:]
 	}
 
+	// parse log tag
+	tag, err := loggerutils.ParseLogTag(ctx, "")
```

Comment:
Looks like it was [inconsistent already](https://github.com/docker/docker/blob/a31435c8a2c4560b6e1a0e84450c45490fa587be/daemon/logger/gelf/gelf.go#L58-L62)